spec/board_spec.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/board'
+require './lib/game'
```

Comment:
### [OPTIONAL]

- [ ] Use require_relative instead of require to link to files in your directory

Reply:
you need it to be 'require' otherwise it doesn't work